images/scripts/fedora.setup

```diff
@@ -34,7 +34,7 @@ PackageKit \
 pcp \
 pcp-libs \
 python3-tracer \
-qemu-system-x86-core \
+qemu-kvm-core \
```

Comment:
What is the motivation for this change in this specific image-refresh?

Reply:
It's unrelated to the boot time fix of course, but I'll have to do this change anyway. So instead of going through all fedora image refreshes twice, it's more efficient to just rebuild them once. It doesn't cause any regression, and unblocks the rpm dependency cleanup.